proposals/61:lo-bundle-cost/Makefile

```diff
@@ -0,0 +1,13 @@
+
+lower-bundle-cost.json:
+	jq '.tx.body.messages[0].content | .changes[0].value |= fromjson' tx-61.json >$@
+
+hash=6EA7B46DFDD0973338BB50781B33D651FE0E8EE8ABF4D7BCB1BF8235334AAFE0
+tx-61.json:
+	agd --node $(ARCHIVE_NODE) query tx $(hash) -o json >$@
+
+clean:
+	rm -f lower-bundle-cost.json
+
+realclean: clean
```

Comment:
why two cleans? please comment

for naming, it's odd to imply that the other isn't real. consider `clean-all` 

but I'd prefer to have `lower-bundle-cost.json` be produced in the build, in which case there'd be only one target and one clean here.

Reply:
`realclean` vs `clean` is a [gnu convention](https://web.mit.edu/gnu/doc/html/make_14.html)

The difference in this case is: re-fetching from the network is expensive / wasteful while I was rapidly iterating on the `jq` step.

Will comment...